foxpuppet/api/windows.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Everything was moved because I wanted to be able to extend foxpuppet.py in some of the other classes, to have access to it. It felt, to me, that it made sense to not have anything on the same 'level' as foxpuppet.py for OOP sake.